python/rikai/spark/functions/vision.py

```diff
@@ -47,9 +49,17 @@
 
 
 @udf(returnType=ImageType())
-def image(uri: str) -> Image:
-    """Build an :py:class:`Image` from a URI."""
-    return Image(uri)
+def image(image: Union[bytes, bytearray, IOBase, str, Path]) -> Image:
```

Comment:
1. i would call the function `to_image` instead?
2. the input argument call it `image_data` or something?
3. what produces the IOBase / bytes / bytearray ? does reading from Spark image data source produce all 3?

Reply:
updated the naming conventions, modeled the input types from [here](https://github.com/eto-ai/rikai/blob/4a97e9d9cf4579ceae9efa90332affaab2714430/python/rikai/types/vision.py#L55) but Spark Images keeps `bytes` in the `Image.data` column.

As VideoCapture extracts numpy arrays, I can update `video_to_frames` with this [PR](https://github.com/smellslikeml/rikai/tree/ffmpeg-load-video) to avoid decompression.